src/views/GameTemplates/GameTemplates.scss

```diff
@@ -0,0 +1,55 @@
+@import "../../index";
+
+.page-container {
+    margin-left: 40px;
+    margin-right: 40px;
+}
+
+.template-header {
+    display: grid;
+    width: 100%;
+    grid-template-columns: 1fr 1fr 1fr;
+    color: $color-white;
+
+    .new-template-button, h1, .input-field {
+        align-self: center;
+    }
+
+    .new-template-button {
+        justify-self: center;
+    }
+
+    h1 {
+        font-size: 3rem;
+    }
+
+    .input-field {
+        input {
+            height: 50px;
+            width: 250px;
+            font-size: 1rem;
+            border: round;
```

Comment:
The `border: round` rule here doesn't seem to make any difference, and it's a producing a CSS warning. What is it supposed to be doing?

Reply:
I'm not getting a warning when I run it with this. It's supposed to change it from a sharp white border to a rounded one but I was never able to get it to work. I'll just remove it since it isn't doing anything anyway.